src/test/java/nextstep/subway/unit/LineTest.java

```diff
@@ -1,17 +1,165 @@
 package nextstep.subway.unit;
 
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.Section;
+import nextstep.subway.domain.Station;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
 class LineTest {
+
+    private Station 강남역;
+    private Station 역삼역;
+    private Station 선릉역;
+    private Station 잠실역;
+    private Long 강남역삼구간거리;
+    private Long 역삼선릉구간거리;
+    private Long 선릉잠실구간거리;
+    private Line 이호선;
+    private Section 강남역삼구간;
+    private Section 역삼선릉구간;
+    private Section 선릉잠실구간;
+
+    @BeforeEach
+    public void setGivenData() {
+        //given
+        강남역 = Station.builder()
+                .id(1L)
+                .name("강남역")
+                .build();
+        역삼역 = Station.builder()
+                .id(2L)
+                .name("역삼역")
+                .build();
+        선릉역 = Station.builder()
+                .id(3L)
+                .name("선릉역")
+                .build();
+        잠실역 = Station.builder()
+                .id(4L)
+                .name("잠실역")
+                .build();
+
+        이호선 = new Line("2호선","Green");
+        강남역삼구간거리 = 5L;
+        역삼선릉구간거리 = 20L;
+        선릉잠실구간거리 = 10L;
+
+        강남역삼구간 = new Section(이호선, 강남역, 역삼역, 강남역삼구간거리);
+        역삼선릉구간 = new Section(이호선, 역삼역, 선릉역, 역삼선릉구간거리);
+        선릉잠실구간= new Section(이호선, 선릉역, 잠실역, 선릉잠실구간거리);
+
+    }
+
+    @DisplayName("라인의 첫 지하철 구간을 등록할 경우.")
```

Comment:
'새로운 역을 상행 종점으로 등록할 경우' 와 '새로운 역을 하행 종점으로 등록할 경우' 를 동시에 테스트하신 것 같은데 
테스트를 2개로 나누거나
```suggestion
    @DisplayName("새로운 역을 상행 종점으로 등록하고 새로운 역을 하행 종점으로 등록할 경우")
```
이렇게 구체적으로 명시하면 좋을 것 같아요:)